src/App.js

```diff
@@ -0,0 +1,26 @@
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { Route, Routes, BrowserRouter } from 'react-router-dom';
+import { feachData } from './Redux/elementSlices';
+import SearchElement from './component/SearchElement';
+import DetailElement from './component/detailElement';
+
+const App = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(feachData());
+  });
+
+  return (
+    <BrowserRouter>
+      <div>
+        <Routes>
+          <Route path="/" element={<SearchElement />} />
+          <Route path="/detailes/:idAPP" element={<DetailElement />} />
+        </Routes>
+      </div>
```

Comment:
- [ ] Dear @girma336, Also your design is cool there are some minor details missing, for the typography, I see you have not set any font face and we need to follow the design guideline to customize the `font face`, `size`, and `weight`. Please update these changes to make your design even better.
- [ ] For the layout: I see you add an image tag, But I would suggest making it a background image for each element to align the composition close to the given design, and you could remove/modify the space between elements to make space between elements as shown in the given design. 

### Figma design 
![Screenshot 2023-01-19 013812](https://user-images.githubusercontent.com/69077061/213310690-2d4db827-c83e-4ed6-9913-8fdaeeab6d68.png)

### Your design
![Screenshot 2023-01-19 013834](https://user-images.githubusercontent.com/69077061/213310698-265005cd-0b14-4b37-8cdd-7f617fb2c727.png)